spec/catalog.yaml

```diff
@@ -166,6 +467,21 @@ components:
             "properties": { "created_at": "1452120468" }
           }
 
+    CreateTableResponse:
+      type: object
+      required:
+        - name
+      properties:
+        name: { type: string }
+        properties:
```

Comment:
Just a bit, we might want to make the terminology consistent. In Database we used options but here we are using properties. In general I think properties is a bit more preferable, because there are SQL key words like DBPROPERTIES, TBLPROPERTIES. And it gives a feeling that it is intrinsic to the object. What do your think?

Reply:
About the `option`, do you mean `write_option`?

Reply:
Also in CreateNamespaceRequest

Reply:
The trade-off I think, is that if we want to align with the naming convention of lance/lancedb. It uses some `options` in their public APIs. I am open about it.